src/test/java/uk/gov/hmcts/juror/api/moj/service/trial/TrialServiceImplTest.java

```diff
@@ -307,15 +308,12 @@ void testReturnJuryConfirmAttendance(String checkInTime) {
         trialService.returnJury(payload, trialNumber, "415",
             createReturnJuryDto(false, "09:00", "10:00"));
 
+        ArgumentCaptor<Appearance> appearanceArgumentCaptor = ArgumentCaptor.forClass(Appearance.class);
+
```

Comment:
doesn't look like this has been used in this method - if you re-instate the below verify() call you can capture and assert the value set or the flag